docs/references.rst

```diff
@@ -3,7 +3,7 @@
 
        <style type="text/css">
        .label {
-       }    
```

Comment:
Sorry, about removing whitespace. I just couldn't resist 🙈

Reply:
sure! there should be no whitespace here. maybe some relict from when this was all markdown and three whitespaces meant a line break